context/context.go

```diff
@@ -17,8 +17,8 @@ func New(rootDir string) *Context {
 		rootDirectory:                rootDir,
 		rulesDirectory:               rootDir + "/rules",
 		utilDirectory:                rootDir + "/util",
```

Comment:
```suggestion
```

Reply:
Thanks. Removed directory from the struct also and associated methods.